pkg/apis/flyteworkflow/v1alpha1/iface.go

```diff
@@ -318,6 +318,7 @@ type ExecutableNode interface {
 	GetRetryStrategy() *RetryStrategy
 	GetExecutionDeadline() *time.Duration
 	GetActiveDeadline() *time.Duration
+	IsInterruptible() *bool
```

Comment:
should this be a pointer? Doesnt absence of this flag automatically imply a `false`?

Reply:
you can always omitempty, so you dont need to make it a pointer

Reply:
Required Behavior: User defines interruptible for a WF which means all the compute from that WF is interruptible. However, this behavior can be overridden by individual nodes.
WF: 
   bool interruptible
      Default value is False: absence of this parameter defaults to false.
Node: 
   bool *interruptible
      In some cases users may actually want to override the interruptible behavior for the node to false. Bool will not work as we will not be able to differentiate absence from false because absence means “use the value defined for the WF, I don’t wish to override"

Reply:
You can also make it an enum... this works fine though..

Reply:
Yes, I am fine either ways. Any hard preference?